target/linux/ath79/image/mikrotik.mk

```diff
@@ -11,6 +11,18 @@ define Device/mikrotik_routerboard-493g
 endef
 TARGET_DEVICES += mikrotik_routerboard-493g
 
+define Device/mikrotik_routerboard-921gs-5hpacd-15s
+  $(Device/mikrotik)
+  SOC := qca9558
+  DEVICE_MODEL := RouterBOARD 921GS-5HPacD-15s
```

Comment:
On [MikroTik's page](https://mikrotik.com/product/RB921GS-5HPacD-15S), the device is called "mANTBox 15s". We could, maybe, use this as the name, plus the product code "RB921GS-5HPacD-15S" in brackets, i.e., `mANTBox 15s (RB921GS-5HPacD-15S)`. "mANTBox" is easier to read and pronounce, and clearer than 921GS-5HPacD-15s.

Thus, [this one](https://github.com/openwrt/openwrt/pull/3140/commits/d3b63682044d12f6e4cf40d444ef2c20c88bcac6#diff-0b7c4fe8bca5c87d97e92ea6f4757fc9R41) should be changed.

Reply:
Just had a look on some other boards, eems the naming is inconsistent anyway. "Mikrotik hEX" is named "MikroTik RB750Gr3" which is the productID. IMHO making this unique should be put in a separate issue anyway.

Reply:
I think that’s another point for going with combined images. At least per platform. My 2c :)

Reply:
@rogerpueyo  according to "RouterBOARD wAP G-5HacT2HnD (wAP AC)" I changed the name to "RouterBOARD 921GS-5HPacD-15s (mANTBox 15s)"

Reply:
> I think that’s another point for going with combined images. At least per platform. My 2c :)

Probably there will be one day an easy to maintain solution for combined images ...